docs/JLBP-0016.md

```diff
@@ -5,8 +5,7 @@ permalink: /JLBP-16
 ---
 # Ensure upper version alignment of dependencies for consumers
 
-Upper version alignment means that the version of each dependency 
-added to the classpath is the highest version in the dependency tree.
```

Comment:
This sentence was defining the term "upper version alignment" and the next sentence was referring to it. Removing the definition breaks the sequence.

Reply:
Definitions belong in the glossary. I think this is clear enough in context, and now it affirmatively recommends the practice rather than simply defining it.